<launch>
    <node name="get_total_pose" pkg="ur_trajectory_follower" type="get_total_Pose.py" output="screen">
        <!-- Topic Remappings -->
        <remap from="world_T_base" to="mir_pose_simple"/>
        <remap from="base_T_ee" to="UR10_r/tcp_pose"/>
        <remap from="world_T_ee" to="global_nozzle_pose"/> <!-- publisher -->

        <param name="world_frame" value="map"/> <!-- for publisher -->
    </node>
</launch>